k8s-openapi-codegen-common/templates/int_or_string.rs

```diff
@@ -65,4 +67,4 @@ impl {local}serde::Serialize for {type_name} {{
             {type_name}::String(s) => s.serialize(serializer),
         }}
     }}
-}}
```

Comment:
Revert these whitespace changes. You can see the whitespace errors this has introduced in the generated code.

Reply:
What kind of text editor are you using? [kakoune can't do it](https://github.com/mawww/kakoune/issues/2147), `nano` can't do it. `gedit` can't do it, `vim` can't do it ...

I'll start to look for hex editor ...

Reply:
Hex editor helped. :D